OperatingSystem-Services/Service-SOC/BasicDocumentation/WazuhDashPassChange.md

```diff
@@ -0,0 +1,74 @@
+# Wazuh Dashbaord Password Change DOCKER
```

Comment:
Maybe move docker to the front `Docker - Changing The Wazuh Dashboard Password` Or something

Reply:
Typo in the first line :(

Reply:
Yea I can't read

> Typo in the first line :(